web/src/main/java/com/graphhopper/http/GraphHopperWeb.java

```diff
@@ -152,7 +154,9 @@ public static PathWrapper createPathWrapper(JSONObject path,
 
                     // The translation is done from the routing service so just use the provided string
                     // instead of creating a combination with sign and name etc
-                    instr.setUseRawName();
+                    // However, you can translate it on the client side by passing translate_insructions=false
```

Comment:
Sorry, I think what I mean is that translate_instruction or 'translate it on the client side' is confusing. We should name it what it does, not what it is supposed to be useful for :)

So what it does IMO is that it returns the raw street name instead of a turn instruction in the name (so that one would need translation on the client side).